cli/login.go

```diff
@@ -353,6 +374,12 @@ func (r *RootCmd) login() *serpent.Command {
 			Description: "Specifies a username to use if creating the first user for the deployment.",
 			Value:       serpent.StringOf(&username),
 		},
+		{
+			Flag:        "first-user-name",
+			Env:         "CODER_FIRST_USER_NAME",
```

Comment:
Nit: should we call this "full name"? `name` and `username` will lead to some confusion, I'd bet.

Reply:
Done!